### ISSUE TITLE
replace oboe by trace view

### ISSUE THREAD

**Issue by joel1di1** [joel1di1/martigua2#137]
details: warning message during gem installation

```
AppNeta TraceView

Note that this will be the final release for the oboe gem.  The gem
will be renamed to 'traceview' and all future updates will be made
to that other gem.

Please update your Gemfiles by replacing the 'oboe' gem with the
'traceview' gem.

Gemfile:
XXX   gem 'oboe'
-->   gem 'traceview'

Any Oboe specific calls will still work as usual in the new gem
although a single deprecation warning will be output on the
first use call.

We will support these legacy calls for a minimum of 1 year but
possibly indefinitely.

Oboe calls can be updated to use the TraceView module instead:

  Oboe::Config           --> TraceView::Config
  Oboe::API              --> TraceView::API
  OboeMethodProfiling    --> TraceViewMethodProfiling
  Oboe::Context          --> TraceView::Context

Thanks for using TraceView!  Happy tracing!

Best,
The TraceView Team

tldr; The gem is being renamed to traceview.  Update your Gemfiles!
```
